extra-dev/packages/coq-mathcomp-real-closed/coq-mathcomp-real-closed.dev/opam

```diff
@@ -17,7 +17,7 @@ order theory of real closed field, through quantifier elimination."""
 build: [make "-j%{jobs}%" ]
 install: [make "install"]
 depends: [
-  "coq" { (>= "8.10" & < "8.14~") | (= "dev") }
+  "coq" { (>= "8.10" & < "8.16~") | (= "dev") }
   "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.13~") | (= "dev")  }
   "coq-mathcomp-algebra" { (>= "1.12.0" & < "1.13~") | (= "dev")  }
   "coq-mathcomp-field" { (>= "1.12.0" & < "1.13~") | (= "dev")  }
```

Comment:
Although I couldn't find previous discussions, I thought I should rather remove these upper bounds in `extra-dev`. If so, I will remove them.

Reply:
Yes, the preferred style for `extra-dev` packages is to drop the upper bound, since this means you don't have to update them as often, and still include `dev`.

Reply:
Thanks for the confirmation. Done.